composer.json

```diff
@@ -18,7 +18,7 @@
         "fast-install" : [
             "php scripts/check.php",
             "php app/console doctrine:database:create --if-not-exists",
-            "php bin/doctrine dbal:import claroline.sql -vvv",
+            "php bin/vendor/doctrine dbal:import claroline.sql -vvv",
```

Comment:
My bad, its `vendor/bin/doctrine` (https://getcomposer.org/doc/articles/vendor-binaries.md#vendor-binaries-and-the-vendor-bin-directory).